Library/ViewModels/RewardPledgeViewModelTests.swift

```diff
@@ -22,7 +22,7 @@ private let shippingRules = locations
     .template
       |> ShippingRule.lens.location .~ location
       |> ShippingRule.lens.cost .~ Double(idx + 1)
-}
+} ||> ShippingRule.lens.location..Location.lens.localizedName %~ { "Local " + $0 }
```

Comment:
Doesn't this read slightly better as:
```swift
private let shippingRules = locations
  .enumerated()
  .map { idx, location in
    .template
      |> ShippingRule.lens.location .~ location
      |> ShippingRule.lens.cost .~ Double(idx + 1)
  }
  ||> ShippingRule.lens.location..Location.lens.localizedName %~ { "Local " + $0 }
```
🤔 Dunno, maybe not.

Reply:
Np! Maybe afterwards since we wanna cut 3.7.1 soon to address these issues.